tests/language/abstract_beats_arguments2_test.dart

```diff
@@ -1,14 +0,0 @@
-// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This being dropped because it is testing a static type warning and that is covered elsewhere?